### PR TITLE
Add a linter for Semantic Line Breaks in Markdown files

### LINKED ISSUE #396: Add a linter for Semantic Line Breaks in Markdown files
We should find and use a good linter for https://sembr.org. Most importantly, it should check SemBr's point 4.

### PR DESCRIPTION
**Pull request by GrandShow**
# Description

[Textlint docs](https://github.com/textlint/textlint)
[Textlint rule one-sentence-per-line docs](https://github.com/IQTLabs/textlint-rule-one-sentence-per-line#readme)

This PR closes #396.

### CODE REVIEW

**Comment by AlekSi** [comment]
@GrandShow Thank you! Please sign our CLA by clicking that button in the message above. And please check that all commits use correct email

File: build/deps/markdownlint.Dockerfile [hunk 1]
```diff
@@ -1 +1,4 @@
 FROM ghcr.io/igorshubovych/markdownlint-cli:v0.32.1
+ENTRYPOINT [""]
+
+RUN npm install --location=global textlint textlint-rule-one-sentence-per-line
```
Comment (AlekSi) [inline build/deps/markdownlint.Dockerfile:4]: Let's create a separate Dockerfile for that linter. Is there a pre-built Docker image we could use?

**Review by AlekSi — CHANGES_REQUESTED** [review]

File: build/deps/markdownlint.Dockerfile (hunk 1 above)
Comment (GrandShow) [reply build/deps/markdownlint.Dockerfile:4]: Textlint doesn't have Docker image, but we can create our own one and install only rules which we need. I will do it now

Comment (AlekSi) [reply build/deps/markdownlint.Dockerfile:4]: Also, https://github.com/IQTLabs/textlint-rule-one-sentence-per-line (btw, check out [this issue](https://github.com/IQTLabs/textlint-rule-one-sentence-per-line/issues/7) ;)) is archived. Is there some alternative?

**Comment by AlekSi** [comment]
@GrandShow please re-request review if it is ready

File: build/deps/markdownlint.Dockerfile (hunk 1 above)
Comment (GrandShow) [reply build/deps/markdownlint.Dockerfile]: @AlekSi I've checked some alternative solutions, in my opinion [this one](https://github.com/bobheadxi/readable) could be good, but my try to make it work with docker wasn't successful.

File: Taskfile.yml [hunk 2]
```diff
@@ -247,6 +247,7 @@ tasks:
     desc: "Format and lint documentation"
     cmds:
       - docker-compose run --rm markdownlint --fix --dot --ignore CHANGELOG.md '**/*.md'
+      - docker-compose run --rm textlint --fix --rule one-sentence-per-line '*.md'
```
Comment (AlekSi) [inline Taskfile.yml:250]: Should we use `**/*.md` to also check files in the `.github` directory?

File: build/deps/textlint.Dockerfile [hunk 3]
```diff
@@ -0,0 +1,7 @@
+FROM node:lts-alpine
```
Comment (AlekSi) [inline build/deps/textlint.Dockerfile:1]: Let's use a fixed tag with a version. Dependabot should be able to send updates for it.

File: build/deps/textlint.Dockerfile [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+
+RUN npm install --location=global textlint textlint-rule-one-sentence-per-line
```
Comment (AlekSi) [inline build/deps/textlint.Dockerfile:3]: Let's use packages.json with exact versions for Dependabot too

**Review by AlekSi — CHANGES_REQUESTED** [review]

File: Taskfile.yml (hunk 2 above)
Comment (GrandShow) [reply Taskfile.yml]: Thank you, fixed

File: build/deps/textlint.Dockerfile (hunk 3 above)
Comment (GrandShow) [reply build/deps/textlint.Dockerfile]: Thank you, fixed

File: build/deps/textlint.Dockerfile (hunk 4 above)
Comment (GrandShow) [reply build/deps/textlint.Dockerfile]: Thank you, fixed

**Review by AlekSi — APPROVED** [review]

**Pull request merged**